proposals/description/fip_77.ts

```diff
@@ -0,0 +1,31 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_77: ProposalDescription = {
+  title: 'FIP-77: Withdraw BentoBox',
+  commands: [
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'set(uint256,uint120,address,bool)',
```

Comment:
you can use `resetRewards(uint256)` with the poolId, but I guess it doesn't matter the result is the same

Reply:
Can't guardian only